libraries/freertos_plus/standard/freertos_plus_tcp/include/FreeRTOSIPConfigDefaults.h

```diff
@@ -314,7 +314,11 @@ from the FreeRTOSIPConfig.h configuration header file. */
 #endif
 
 #ifndef ipconfigALLOW_SOCKET_SEND_WITHOUT_BIND
-	#define ipconfigALLOW_SOCKET_SEND_WITHOUT_BIND 1
+	#define ipconfigALLOW_SOCKET_SEND_WITHOUT_BIND	1
+#endif
+
+#ifndef ipconfigIP_PASS_PACKETS_WITH_IP_OPTIONS
+	#define ipconfigIP_PASS_PACKETS_WITH_IP_OPTIONS	1
```

Comment:
Where is this documented?  Why would someone set `ipconfigIP_PASS_PACKETS_WITH_IP_OPTIONS==0`?

Reply:
When the FreeRTOS+TCP stack see IP-options, it will call `memmove()` to get rid of the extra data.
Some developers may want to refuse any packet that contains IP-options. This option make this possible.